### ISSUE TITLE
Support other Ruby versions than 2.5.1

### ISSUE THREAD

**Issue by skateman** [hellostealth/stealth#71]
The `Gemfile` generated by `stealth new <app>` hardcodes ruby version to be `2.5.1`, however, the gem itself doesn't specify `2.5.1` as the `required_ruby_version` in the gemspec.

I was able to install the gem on `2.4.4p296` and then got the following error when running `bundle install`:
```
Your Ruby version is 2.4.4, but your Gemfile specified 2.5.1
```

I have three suggestions to fix this:
* Drop the ruby version from the `Gemfile`
* Set the `required_ruby_version` in the gemspec
* Use a template `Gemfile` and set the ruby version dynamically when generating a new app

What do you think?

**Comment by mgomes** [comment]
It's actually not required. We test against Ruby `2.4.x` and Ruby `2.5.x`. This was just oversight in the `Gemfile`. Will be fixed in `1.0.2`.

**Comment by mgomes** [comment]
Fixed in 1.0.2.
